runtime/runtime-params-estimator/src/main.rs

```diff
@@ -137,11 +140,17 @@ fn main() -> anyhow::Result<()> {
         return main_docker(&state_dump_path, cli_args.full, cli_args.docker_shell);
     }
 
+    if let Some(compare_to) = cli_args.compare_to {
+        let baseline = cli_args.costs_file.unwrap();
+
+        let compare_to = read_costs_table(&compare_to)?;
+        let baseline = read_costs_table(&baseline)?;
+        println!("{}", baseline.diff(&compare_to));
+        return Ok(());
+    }
+
     if let Some(path) = cli_args.costs_file {
-        let cost_table = fs::read_to_string(&path)
-            .ok()
-            .and_then(|it| it.parse::<CostTable>().ok())
-            .with_context(|| format!("Failed to parse {}", path.display()))?;
+        let cost_table = read_costs_table(&path);
```

Comment:
I think this Result needs to be unpacked
```suggestion
        let cost_table = read_costs_table(&path)?;
```